lesson_06/task06_3/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+/*var (
+	// our message
+	message = "Hello there!"
+)*/
+
+type Page struct {
+	Title string
+	Body  []byte
+}
+
+func loadPage(title string) (*Page, error) {
+	filename := title + ".html"
+	body, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return nil, err
+	}
+	return &Page{Title: title, Body: body}, nil
+}
+
+func handler(w http.ResponseWriter, r *http.Request) {
+	// send Not found in such case
+	if r.URL.Path != "/" {
+		http.NotFound(w, r)
+		return
+	}
+	if r.Method == "GET" {
```

Comment:
We have special consts in http package - http.MethodGet. It is better to use it to avoid unexpected typos or mistakes